workflow-core/src/main/java/com/squareup/workflow/WorkflowOperators.kt

```diff
@@ -18,24 +18,22 @@ package com.squareup.workflow
 import kotlinx.coroutines.experimental.CoroutineScope
 import kotlinx.coroutines.experimental.Deferred
 import kotlinx.coroutines.experimental.Dispatchers.Unconfined
-import kotlinx.coroutines.experimental.GlobalScope
 import kotlinx.coroutines.experimental.Job
 import kotlinx.coroutines.experimental.async
 import kotlinx.coroutines.experimental.channels.Channel.Factory.CONFLATED
 import kotlinx.coroutines.experimental.channels.ReceiveChannel
-import kotlinx.coroutines.experimental.channels.consume
 import kotlinx.coroutines.experimental.channels.consumeEach
 import kotlinx.coroutines.experimental.channels.produce
 import kotlinx.coroutines.experimental.channels.toChannel
-import kotlinx.coroutines.experimental.selects.whileSelect
+import kotlinx.coroutines.experimental.launch
 import kotlin.coroutines.experimental.CoroutineContext
 
 /**
- * [CoroutineContext] used by [Workflow] operators below.
+ * Defines the [CoroutineContext] used by [Workflow] operators below.
  *
  * See this module's README for an explanation of why Unconfined is used.
  */
-private val operatorContext: CoroutineContext = Unconfined
+private val operatorScope = CoroutineScope(Unconfined)
```

Comment:
What's the significance of the change from `GlobalScope` to our own `CoroutineScope()`?

Reply:
Just changes how we use it for the builders below.
```kotlin
GlobalScope.foo(operatorContext) {
```
becomes
```
operatorScope.foo {
```

Reply:
Right, but we're building a scope from scratch instead of using that magic singleton. `GlobalScope` is the same as `CoroutineScope()`?

Reply:
Answer appears to be yes, since we were always overriding its context anyway.

Reply:
`GlobalScope` is just a scope with an empty context.